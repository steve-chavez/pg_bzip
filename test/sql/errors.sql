create extension if not exists bzip;
\echo

select bzcat(pg_read_binary_file('./damaged_all_movies.csv.bz2'));
\echo

select bzcat(pg_read_binary_file('./empty_file'));
\echo

select bzcat(pg_read_binary_file('./wrong_header_all_movies.csv.bz2'));
