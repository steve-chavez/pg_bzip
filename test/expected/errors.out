create extension if not exists bzip;
NOTICE:  extension "bzip" already exists, skipping
\echo

select bzcat(pg_read_binary_file('./damaged_all_movies.csv.bz2'));
ERROR:  bzip2 decompression failed: compressed data ends unexpectedly
\echo

select bzcat(pg_read_binary_file('./empty_file'));
ERROR:  bzip2 decompression failed: compressed data ends unexpectedly
\echo

select bzcat(pg_read_binary_file('./wrong_header_all_movies.csv.bz2'));
ERROR:  bzip2 decompression failed: data is not in bzip2 format
